ams/server/src/main/java/com/netease/arctic/server/dashboard/PaimonTableDescriptor.java

```diff
@@ -136,12 +148,90 @@ public ServerTableMeta getTableDetail(AmoroTable<?> amoroTable) {
 
   @Override
   public List<TransactionsOfTable> getTransactions(AmoroTable<?> amoroTable) {
-    throw new UnsupportedOperationException();
+    FileStoreTable table = getTable(amoroTable);
+    List<TransactionsOfTable> transactionsOfTables = new ArrayList<>();
+    Iterator<Snapshot> snapshots;
+    try {
+      snapshots = table.snapshotManager().snapshots();
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+    AbstractFileStore<?> store = (AbstractFileStore<?>) table.store();
+    while (snapshots.hasNext()) {
+      Snapshot snapshot = snapshots.next();
+      if (snapshot.commitKind() == Snapshot.CommitKind.COMPACT) {
+        continue;
+      }
+      Map<String, String> summary = new HashMap<>();
+      summary.put("commitUser", snapshot.commitUser());
+      summary.put("commitIdentifier", String.valueOf(snapshot.commitIdentifier()));
+      if (snapshot.watermark() != null) {
+        summary.put("watermark", String.valueOf(snapshot.watermark()));
+      }
+
+      //record number
+      if (snapshot.totalRecordCount() != null) {
+        summary.put("total-records", String.valueOf(snapshot.totalRecordCount()));
+      }
+      if (snapshot.deltaRecordCount() != null) {
+        summary.put("delta-records", String.valueOf(snapshot.deltaRecordCount()));
+      }
+      if (snapshot.changelogRecordCount() != null) {
+        summary.put("changelog-records", String.valueOf(snapshot.changelogRecordCount()));
+      }
+
+      //file number
+      TransactionsOfTable deltaTransactionsOfTable = manifestListInfo(store, snapshot, (m, s) -> s.deltaManifests(m));
+      int deltaFileCount = deltaTransactionsOfTable.getFileCount();
+      int dataFileCount = manifestListInfo(store, snapshot, (m, s) -> s.dataManifests(m)).getFileCount();
+      int changeLogFileCount = manifestListInfo(store, snapshot, (m, s) -> s.changelogManifests(m)).getFileCount();
+      summary.put("delta-files", String.valueOf(deltaFileCount));
+      summary.put("data-files", String.valueOf(dataFileCount));
+      summary.put("changelogs", String.valueOf(changeLogFileCount));
```

Comment:
Is `changelogs` related to records or files? How does the front-end know which chart to depict it in?